sorc/chgres_cube.fd/search_util.F90

```diff
@@ -215,27 +215,47 @@ subroutine search (field, mask, idim, jdim, tile, field_num, latitude, terrain_l
 
  end subroutine search
 
-!> Set sst values based on latitude.
+!> Set default sst values based on latitude.
 !!
-!! @param latitude latitude input
-!! @param sst sst guess value to be set
+!! Based loosely on the average annual SST
+!! values from ./fix_am/cfs_oi2sst1x1monclim19822001.grb
+!!
+!! The temperature in the polar and tropical regions
+!! is set to 273.16/300 Kelvin. Polar regions are
+!! poleward of 60 degrees. Tropical regions are within
+!! 30 degrees of the equator. In mid-latitudes, a
+!! linear change with latitude is used. 
+!!
+!! @param [in] latitude Latitude in degrees
+!! @param [out] sst Default SST in Kelvin
 !! @author George Gayno NCEP/EMC
  subroutine sst_guess(latitude, sst)
 
  use esmf
 
  implicit none
 
+ real(esmf_kind_r8), parameter :: sst_polar_in_kelvin = 273.16 !< Default SST in polar
```

Comment:
The usual convention is to name parameter variables in all caps, for example SST_POLAR_IN_KELVIN.

Of course since fortran ignores case anyway, this is purely a convention, and not a perfect one. However, it does help make clear in the code what part of a formula is a constant and what parts are variables.